lambdas/service/app.py

```diff
@@ -294,9 +285,7 @@ def get_data_pie():
         # Returning a single response if <file_id> request form is used
         return response
     except BadArgumentException as bae:
-        response = dict(error=bae.message)
-        response.status_code = 400
-        return response
+        raise BadArgumentException(bae.message)
```

Comment:
Why do we catch BadArgumentException if we then raise another exception of the same type?

Reply:
Made a mistake. Meant to type `BadRequestError`